script/UpdateLtvLeveragedEthMainnet.s.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.13;
+
+import {scWETH} from "../src/steth/scWETH.sol";
+import {IPool} from "aave-v3/interfaces/IPool.sol";
+import {Constants as C} from "../src/lib/Constants.sol";
+import {DataTypes} from "aave-v3/protocol/libraries/types/DataTypes.sol";
+import {WadRayMath} from "aave-v3/protocol/libraries/math/WadRayMath.sol";
+
+import "forge-std/Script.sol";
+
+contract UpdateLtvLeveragedEthMainnet is Script {
+    using WadRayMath for uint128;
+
+    address keeper = vm.envAddress("KEEPER");
+    scWETH vault = scWETH(payable(vm.envAddress("scWETH")));
+    IPool aaveV3Pool = IPool(C.AAVE_V3_POOL);
+
+    uint256 bestLtv = 0.8e18;
+    uint256 lowestLtv = 1e8;
+
+    function run() external {
+        // get Lido Interest (with 18 decimals)
+        uint256 lidoInterest = 0; // TODO
```

Comment:
TODO should be addressed before merge

Reply:
The main idea is to not merge this PR but port the code here to hardhat in our backend repo.